Core/GDCore/IDE/Events/ExpressionTypeFinder.h

```diff
@@ -72,7 +72,7 @@ class GD_CORE_API ExpressionTypeFinder : public ExpressionParser2NodeWorker {
         child(nullptr) {};
 
   const gd::String &GetType() {
-    return gd::ParameterMetadata::GetExpressionValueType(type);
+    return gd::ValueTypeMetadata::GetExpressionPrimitiveValueType(type);
```

Comment:
This doesn't change anything. It's just to navigate more easily without going through deprecated functions.